themes/StarterTheme/templates/catalog/products.tpl

```diff
@@ -0,0 +1,17 @@
+{if $products|count}
+  <section id="products">
+    <h1>{l s='Products'}</h1>
```

Comment:
There must be only one `h1` in a document, having more than one is a SEO mistake. Sections must have a heading element, but it doesn't have to be `h1` (can be `h2`, `h3`, etc.)